src/rust/engine/cargo_build_shim.sh

```diff
@@ -6,7 +6,8 @@
 # from the rule's env and the `extra_env_vars`.
 
 PYTHON_PATH="${CHROOT}/3rdparty/tools/python3/python/bin"
-export PATH="$PATH:$PYTHON_PATH"
+PROTOC_PATH=${CHROOT}/3rdparty/tools/protoc/protoc/bin
```

Comment:
```suggestion
PROTOC_PATH="${CHROOT}/3rdparty/tools/protoc/protoc/bin"
```

Reply:
She'll scripts don't like me. And I don't like them either

😂